tests/integration/test_pairwise_significance.py

```diff
@@ -499,57 +454,83 @@ def test_ttests_use_unweighted_n_for_variance(self):
         )
         np.testing.assert_array_equal(pairwise_indices, expected_indices)
 
-    def test_cat2_x_mr_itself_pairwise_compare_columns(self):
-        slice_ = Cube(CR.CAT2_X_MR_ITSELF).partitions[0]
+    def test_cat_hs_x_mr_augmented_pairwise_t_test(self):
+        slice_ = Cube(CR.CAT_HS_X_MR_AUGMENTED).partitions[0]
+        actual = slice_.pairwise_significance_tests[0]
+
+        assert slice_.cube_is_mr_by_itself is True
+        np.testing.assert_array_almost_equal(
+            actual.t_stats,
+            [
+                [0.0, 6.22407684, np.nan],
+                [0.0, 0.21986422, np.nan],
+                [0.0, -2.39283195, np.nan],
+                [0.0, -3.83382639, np.nan],
+            ],
+        )
+        np.testing.assert_array_almost_equal(
+            actual.p_vals,
+            [
+                [1.00000000e00, 2.28560104e-09, np.nan],
+                [1.00000000e00, 8.26172768e-01, np.nan],
+                [1.00000000e00, 1.75259091e-02, np.nan],
+                [1.00000000e00, 1.63076135e-04, np.nan],
+            ],
+        )
+
+    def test_cat_x_mr_weighted_augmented(self):
+        slice_ = Cube(CR.CAT_X_MR_WEIGHTED_AUGMENTED).partitions[0]
         actual = slice_.pairwise_significance_tests[1]
 
         assert slice_.cube_is_mr_by_itself is True
         np.testing.assert_array_almost_equal(
             actual.t_stats,
-            [[-1.11043969, 0.0, -38.08479951], [1.95585649, 0.0, -43.33389473]],
+            [[-10.2964264, 0.0, -20.09577285], [5.96350953, 0.0, 24.14335882]],
```

Comment:
can we see the actual proportions being compared, either in a comment or additional insertion?